examples/websites/README.md

```diff
@@ -43,6 +43,8 @@ $ ipfs name publish $SITE_HASH
 Published to <your peer id>: /ipfs/$SITE_HASH
 ```
 
+(Disclaimer: IPNS is not production ready and assets could be loaded from two different resolved hashes when updating your website, leading to outdated/missing assets. Better to use IPFS hash directly to be sure the right asset is being loaded)
+
```

Comment:
Recent change in dht should improve DHT performance but it will be seen in practice.